app/views/browse/index.html.erb

```diff
@@ -0,0 +1,57 @@
+
+<h1>Browse by Call Number</h1>
+
+<h2 class="h4"><%= @shelf_list.list.first.call_number %> to <%= @shelf_list.list.last.call_number %></h2>
+
+<div class="row my-3">
+  <div class="col-6">
+    <%= render Browse::PageNavigation.new(prev_item: @shelf_list.previous,
+                                          next_item: @shelf_list.next,
+                                          length: params[:length]) %>
+  </div>
+  <div class="col-6 text-right">
+    <%= render Browse::PageSizeSelector.new(params: params) %>
+  </div>
+</div>
+
+<table class="table table-striped my-3">
+  <thead>
+    <tr>
+      <th scope="col">Call Number</th>
+      <th scope="col">Title</th>
+      <th scope="col">Location</th>
+      <th scope="col">Publication Info</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @shelf_list.list.map do |shelf_item| %>
+        <% if shelf_item.nearby? && !shelf_item.match? %>
+          <tr class="<%= shelf_item.nearby? ? 'table-primary' : '' %>">
```

Comment:
class here will always be `table-primary`

Reply:
Classic copy paste mistake... 🤦

Reply:
happens to us all...